setup/pattoo_installation.py

```diff
@@ -380,10 +379,11 @@ def main():
             # Import db after pip3 packages are installed
             from _pattoo import db
             db.install()
-            systemd.install(daemon_list=daemon_list,
-                            template_dir=template_dir,
-                            installation_dir=installation_dir,
-                            verbose=args.verbose)
+            if getpass.getuser() == 'root':
```

Comment:
You've repeated this section in the same file. Make it a function for simplification of the code. See whether there are any other similar instances here.